docs/advanced/python.rst

```diff
@@ -0,0 +1,15 @@
+Python
+******
+
+It is possible to use emloop from python, although this approach is discouraged.
+For creating model, dataset, hooks, output directory and main loop you can use following function defined in file `emloop/cli/common.py`:
+
+.. code-block:: python
+    create_output_dir
+    create_dataset
+    create_model
+    create_hooks
+    create_main_loop
+
+All these functions require configuration dictionary, which you can get from standard YAML config file using function `load_yaml` defined in `emloop/utils/yaml.py`.
+Then you can use methods `main_loop.train_by_stream` and `main_loop.evaluate_stream` for training and evaluating one epoch.
```

Comment:
Thank you for mentioning this in the documentation. However, this will surely need more love. There has to be an example, the functions in the list need to link to the API docs and the whole Python API should be directly under `emloop` package or under a dedicated subbackage for simpler usage.